universe/engine.py

```diff
@@ -1,110 +1,101 @@
-from decimal import Decimal
-import random
+from . import components, systems
+from .orders import (Move, CargoTransfer, Scrap, BuildInstallation, Terraform,
+                     BuildStation, BuildShip, LaunchMassPacket)
 
 
-class GameState(object):
+class Entity:
```

Comment:
Thoughts: I think that we should probably add a `__getattr__` method (plus `__setattr__`) that rummages around through each component's `_fields` to see what we allow.  That way doing `my_entity.x` will return `my_entity._data['x']`, but only if there is a component in `my_entity._components` that has an `x` field.